packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/tests/useBlocksStore.test.js

```diff
@@ -897,4 +897,118 @@ describe('useBlocksStore', () => {
     // Check if the cursor is positioned in the new line
     expect(cursorPositionAfterSecondEnter[0]).toEqual(cursorInitialPosition[0] + 1);
   });
+
+  it('disables modifiers when creating a new node with enter key in a paragraph', () => {
```

Comment:
Can you create also a test to check if the modifiers are active when you click enter in between an italic text for example?